pkg/cluster/cluster.go

```diff
@@ -470,9 +467,13 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
 			}
 
 			// Only display add-ons for which the org has quota
-			resourceQuotas.Each(func(resourceQuota *amsv1.ResourceQuota) bool {
-				if addOn.ResourceName() == resourceQuota.ResourceName() {
-					clusterAddOn.Available = float64(resourceQuota.SkuCount()) >= addOn.ResourceCost()
+			quotaCosts.Each(func(quotaCost *amsv1.QuotaCost) bool {
+				relatedResources := quotaCost.RelatedResources()
+				for _, relatedResource := range relatedResources {
+					if relatedResource.ResourceType() == "add-on" &&
+						addOn.ResourceName() == relatedResource.ResourceName() {
+						clusterAddOn.Available = true
```

Comment:
you can add `continue` here to exit the loop. once you know the addon matches, you don't need to continue looping.

Reply:
I suppose you mean to `break`